modules/axios/plugin.js

```diff
@@ -135,6 +135,15 @@ export default (ctx) => {
   });
   <% } %>
 
+  <% if (options.requestInterceptor) { %>
+  // Custom request interceptor
+  const reqInter = <%= options.requestInterceptor %>
+  axios.interceptors.request.use(
+    (config) => reqInter(config, store),
+    (error) => Promise.reject(error)
```

Comment:
I think we can simply omit the second argument letting axios use default error handler